source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -113,10 +115,6 @@ void features.add({
 	exclude: [
 		() => !pageDetect.isForkedRepo()
 	],
-	init() {
-		observeElement([
-			'[data-target="branch-filter-controller.results"]', // Pre "Repository refresh" layout
-			'[data-target="branch-filter.result"]'
-		].join(), init);
-	}
+	waitForDomReady: false,
```

Comment:
Can be dropped because it can't be avoided
```suggestion
```

Reply:
We would gain that the request is sent before dom ready